src/ScanCommand.php

```diff
@@ -49,6 +49,27 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'The maximum number of seconds the request can take',
                 10
+            )
+            ->addOption(
+                'user-agent',
+                'u',
+                InputOption::VALUE_OPTIONAL,
+                'The User Agent to pass for the request',
+                ''
+            )
+            ->addOption(
+                'verify',
```

Comment:
The standard behaviour is that the ssl certificate is checked. Change this option into one that can disable the ssl check.